composition/src/warnings/warnings.ts

```diff
@@ -130,3 +131,27 @@ export function requiresDefinedOnNonEntityFieldWarning(fieldCoords: string, subg
     },
   });
 }
+
+export function consumerInactiveThresholdValueShouldBePositiveWarning(subgraphName: string): Warning {
+  return new Warning({
+    message:
+      `The "consumerInactiveThreshold" argument of type "Int" should be positive and smaller than 2 147 483 647.\n` +
+      `The value has been set to ` +
+      DEFAULT_CONSUMER_INACTIVE_THRESHOLD +
+      `.`,
+    subgraph: {
+      name: subgraphName,
+    },
+  });
+}
+
+export function consumerInactiveThresholdValueShouldNotBeTooBigWarning(subgraphName: string): Warning {
+  return new Warning({
+    message:
+      `The "consumerInactiveThreshold" argument of type "Int" should be positive and smaller than 2 147 483 647.\n` +
+      `The value has been set to 0, that is equivalent to keep the consumer active forever.`,
```

Comment:
The value has been set to 0. This means the consumer will remain indefinitely active until its manual deletion.